### PR TITLE
Don't use RequestsContext

### PR DESCRIPTION
**Pull request by stsewd**
This is kind of deprecated on django 1.11 https://docs.djangoproject.com/en/2.1/releases/1.11/#django-template-backends-django-template-render-prohibits-non-dict-context

The solution s compatible with older versions

Ref https://github.com/rtfd/readthedocs.org/pull/4750

### CODE REVIEW

**Comment by stsewd** [comment]
I think we don't have tests for this, we only have a couple of tests for the forms, not the views. I'll check the coverage

**Comment by stsewd** [comment]
Well, we have only 20% coverage here p:

**Comment by stsewd** [comment]
I'm writing some tests for this, but @davidfischer feel free to merge this if you need to keep going with the django upgrade. I think there are some bugs on the views and it will take some time

**Comment by davidfischer** [comment]
I don't think there's a *huge* rush although the Django upgrade is pretty high up my priority list. You have plenty of time to write tests.

**Comment by stsewd** [comment]
And we have 98%  coverage on `profiles/view.py` now :)

I also deleted an unused view (`create_profile`), I tested with a new user and we don't use it, also it wasn't working before https://readthedocs.org/accounts/create/ anyway.

File: readthedocs/profiles/views.py [hunk 1]
```diff
@@ -2,114 +2,22 @@
...
-        The profile-creation form.
-
-    **Template:**
-
-    ``template_name`` keyword argument, or
-    :template:`profiles/create_profile.html`.
-    """
-    try:
-        profile_obj = request.user.profile
-        return HttpResponseRedirect(reverse('profiles_edit_profile'))
```
Comment (stsewd) [inline readthedocs/profiles/views.py]: Here was a bug, it should be `profiles_profile_edit`

**Comment by humitos** [comment]
> Let me know if we still need `create_profile` to put it back again and fix the bug

I grepped all the repositories and I didn't find that it's used in any place. I think we are safe removing it.

**Comment by humitos** [comment]
@ericholscher I think we can merge this one, can you confirm that the method removed is not needed and it's OK by removing it?

**Comment by ericholscher** [comment]
Yep. I think we should probably just get rid of profiles here at some point, since they are barely used. It's actually a vendoring of the old `django-profiles` app.

**Pull request merged**